src/Razor/src/Microsoft.VisualStudio.RazorExtension/EmbeddedGrammars/aspnetcorerazor.tmLanguage.tmTheme

```diff
@@ -215,7 +215,7 @@
         <key>settings</key>
         <dict>
           <key>vsclassificationtype</key>
-          <string>User Members - Parameters</string>
+          <string>parameter name</string>
```

Comment:
What's the deal with this? Does sound like it might match the scope string better, but I'm curious how you arrived at this change.

Reply:
Good question, so in the %temp% directory there's a `CommonLanguageLog.txt` file that captures exceptions from the CoreEditor when interpreting the various TextMate, langauge-configuraiton.json pieces of the editor (it's how I knew if things were breaking when updating the lang configs) and I noticed this:
```
6/28/2021 10:23:44 AM: Unable to find explicitly specified classification type: User Members - Parameters
```

Reply:
Good catch!